target/linux/ramips/patches-6.12/931-mips-ralink-add-missing-include.patch

```diff
@@ -0,0 +1,25 @@
+From: Mieczyslaw Nalewaj <namiltd@yahoo.com>
```

Comment:
I checked, and like the pci-rt2800 issue this is an upstream issue and should be fixed there.

I'll gladly help you sending in patches upstream. I know it feels intimidating, but it isn't actually that complicated, especially for oneliner fixes.